python/core/auto_generated/qgsapplication.sip.in

```diff
@@ -380,9 +380,19 @@ colors are specified then the ``size`` argument also must be set.
 Returns the path to user's style.
 %End
 
-    static QRegExp shortNameRegExp();
+
+    static QRegExp shortNameRegExp() /Deprecated/;
```

Comment:
This won't compile on qt 6 builds... I'd just remove it entirely and document the API break.

Reply:
@nyalldawson , oh true. I'm definitively +1 for an API break, it's unlikely anyone has been using this function to begin with.

Should I then merely change the return type of shortNameRegExp() to QRegularExpression?

Reply:
No, that'd make identifying breakage harder vs just removing the broken method